configuration/configuration.go

```diff
@@ -38,6 +38,7 @@ const (
 	varConsoleURL                      = "console.url"
 	varOpenshiftCheVersion             = "openshift.che.version"
 	varOpenshiftJenkinsVersion         = "openshift.jenkins.version"
+	varOpenshiftJenkinsRootURL         = "openshift.jenkins.root.url"
```

Comment:
Please follow the naming from https://github.com/chmouel/fabric8-tenant/blob/0d6fadc590037b65564d7acddedb8daa728c5ce4/configuration/configuration.go#L45

Reply:
so something like varTemplateJenkinsRootURL instead? (not sure what is the recommender and if that's what you are pointing at)